tests/unit/SidebarFilterTest.js

```diff
@@ -272,97 +272,6 @@ describe('Sidebar', () => {
                     })
             );
         });
-
-        // NOTE: This is also for #focus mode, should we move this test block?
```

Comment:
I left some context here: https://github.com/Expensify/App/pull/21406/files#r1380809409

After carefully reviewing this test block - my conclusion is that it's the exact same as these lines starting [here](https://github.com/Expensify/App/blob/444a908335bfd9888949718b97dce86c023d59c4/tests/unit/SidebarFilterTest.js#L602-L604).

It must be a mistake as well since there's no reason to test whether reports are or are not appearing in the LHN when in "Most recent" mode. All the reports pretty much show up with some exceptions (that we are not testing).